frontend/packages/console-app/src/components/quick-starts/catalog/__tests__/QuickStartTileDescription.spec.tsx

```diff
@@ -14,10 +14,21 @@ jest.mock('react-i18next', () => {
 
 describe('QuickStartCatalog', () => {
   it('should show prerequisites only if provided', () => {
-    const QuickStartTileDescriptionProps = getQuickStarts()[0].spec;
+    const quickStart = getQuickStarts()[0].spec;
```

Comment:
nit: should we name it to `quickStartSpec` ?

Reply:
Technically its a quick start from the perspective of the current quick start.